airflow/executors/base_executor.py

```diff
@@ -304,6 +304,9 @@ def execute_async(
         """
         raise NotImplementedError()
 
+    def get_task_log(self, ti: TaskInstance, log: str = "") -> None | str | tuple[str, dict[str, bool]]:
```

Comment:
Just noticed that this is missing a doc string. Do you mind adding one that explains what this method does and that expectation is child classes implement it (if desired) and also documenting the params/return values.

Reply:
Done